csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.shared.cs

```diff
@@ -449,7 +449,7 @@ public void AppendExecutionProvider(string providerName, Dictionary<string, stri
         /// <param name="epOptions">Optional options to configure the execution provider. May be null.</param>
         /// <exception cref="ArgumentException">epDevices was empty.</exception>
         /// <see cref="OrtEnv.GetEpDevices" />
-        public void AppendExecutionProvider(OrtEnv env, IReadOnlyList<OrtEpDevice> epDevices, 
+        public void AppendExecutionProvider(OrtEnv env, IReadOnlyList<OrtEpDevice> epDevices,
```

Comment:
Looks like this file only has whitespace differences.

Reply:
The auto linter in my editor did this (it uses our linter settings in the ORT repo).

Reply:
i'll undo it to reduce noise